support/util/imagemetadata_test.go

```diff
@@ -236,7 +236,7 @@ func TestGetDigest(t *testing.T) {
 			pullSecret:     pullSecret,
 			expectedErr:    false,
 			validateCache:  true,
-			expectedDigest: "sha256:dfa54ef35e438b9e71ac5549159074576b6382f95ce1a434088e05fd6b730bc4",
+			expectedDigest: "sha256:fe9080e7b47dd0751812e64eb2510b0444b3706fbbe426c4e9372dd8a77141ee",
```

Comment:
Hey Patrick, I don't know how urgent is this for you, I assume pretty much. For me this is a quick patch that maybe for the urgency, I think make sense to implement now, but this could be solved in 2 ways:
- Modify the test to use a mocked response (It's a unit test and should not need communication with the final registry)
- Put that test under test/integration folder and keep it as it is.

As I said, if this is urgent I will tag it. If not, do you mind update the PR have a generated Mocked response?

Reply:
It's currently blocking all pull requests , so id say it's pretty urgent. However I can try having a look at these suggestions in a subsequent PR if you want